hazelcast/src/main/java/com/hazelcast/jet/core/metrics/MetricTags.java

```diff
@@ -47,17 +47,26 @@ public final class MetricTags {
 
     /**
      * Unique ID of the job (sourcing the metric), example value would be a
-     * numerical (long) ID encoded in a human readable form, like {@code
-     * "2f7f-d88a-4669-6195"}, see {@link Util#idToString(long)})} for
-     * details.
+     * numerical (long) ID encoded in a human-readable form, like
+     * {@code "2f7f-d88a-4669-6195"}.
+     *
+     * @see Util#idToString(long)
```

Comment:
I personally prefer the in-text links, rather than links in a "See Also" section at the bottom. Compare:
![image](https://user-images.githubusercontent.com/9109877/221131551-b224e9b9-3cd4-4bd3-99f6-ffd857f09b9b.png)
vs.
![image](https://user-images.githubusercontent.com/9109877/221131409-2881cfbe-efb1-4194-9f96-cf532d12cf0f.png)